projects/nlp/hello_classification_model.py

```diff
@@ -23,44 +23,55 @@
 helpStr = """
 Simple script that explains how to run classification models.
 
-Example invocations:
+The standard dataset is 10 documents of three-word sentences, each classified
+into one of two categories. Bag of words (BOW) models should be able to
+discriminate them.
+Test data will be a copy of training data plus two additional documents. The
+first is an incorrectly labeled version of a training sample. The second is
+semantically similar to one of the training samples.
+Expected classification accuracy using CIO encoder is 9 out of 10 = 90%
+Expected classification accuracy using keywords encoder is 8 out of 10 = 80%
 
+Example invocations:
 python hello_classification_model.py -m keywords
-python hello_classification_model.py -m docfp
-python hello_classification_model.py -c data/network_configs/sensor_knn.json -m htm -v 2
-python hello_classification_model.py -c data/network_configs/tp_knn.json -m htm
-
+python hello_classification_model.py -m docfp -v 0
+python hello_classification_model.py -m htm \
+  -c data/network_configs/sensor_simple_tp_knn.json \
+  --retina en_associative_64_univ
 """
 
 import argparse
 import copy
+from prettytable import PrettyTable
+from textwrap import TextWrapper
+from tqdm import tqdm
 
 from htmresearch.frameworks.nlp.classification_model import ClassificationModel
 from htmresearch.frameworks.nlp.classify_htm import ClassificationModelHTM
 from htmresearch.frameworks.nlp.model_factory import (
   createModel, getNetworkConfig)
+from htmresearch.support.csv_helper import mapLabelRefs, readCSV
+
+
+wrapper = TextWrapper(width=80)
+
+
+
+def getData(dataPath):
```

Comment:
For the "hello" example file I would prefer to keep the data explicit within the file rather than have a level of indirection and read in another file. Much simpler to understand (you don't need to open and understand a file format and the usage of mapLabelRefs).  Note that you are not saving any lines of code by adding this getData method. It is also harder to understand the explanation comments above because they are abstracted from the data.